Weather Man/Weather_Man/ReportCreator.py

```diff
@@ -0,0 +1,169 @@
+            result_calculator_obj.calculate_avg_highest_temp_in_month(
+                highest_temp_list)
+            result_calculator_obj.calculate_avg_lowest_temp_in_month(
+                lowest_temp_list)
+            result_calculator_obj.calculate_avg_mean_humidity_in_month(
+                mean_humidity_list)
+
+            print(result_calculator_obj.avg_highest_temp_in_month)
+            print(result_calculator_obj.avg_lowest_temp_in_month)
+            print(result_calculator_obj.avg_mean_humidity_in_month)
+
+    def monthly_graph(self, file_name, year, month):  # second test case report generator
+
+        # no of files exits
+        file_paths_list = []
+
+        # counter for loop a complete year
+        file_path = ('{}_{}_{}.txt'.format(
+            file_name, year, month))
+
+        if os.path.isfile(file_path):
+            file_paths_list.append(file_path)
+
+        data_obj = FileReader()
+        if len(file_paths_list) == 0:
+            print("No record exists for this year, Try another year!")
+        else:
+            # filled the data of the year give
+            data_obj.data_filler(file_paths_list)
+
+            # list for only max temperature of all tha days of the year
+            highest_temp_list = []
+            highest_temp_list = data_obj.max_temperature_c
+
+            # list for only min temperature of all tha days of the year
+            lowest_temp_list = []
+            lowest_temp_list = data_obj.min_temperature_c
+
+            result_calculator_obj = ResultCaluculator()
+            result_calculator_obj.draw_monthly_temperature_chart(
+                highest_temp_list, lowest_temp_list)
+
+            print(result_calculator_obj.monthly_temperature_chart)
+
+    def monthly_graph_bonus_task(self, file_name, year, month):  # second test case report generator
+
+        # no of files exits
+        file_paths_list = []
+
+        # counter for loop a complete year
+        file_path = ('{}_{}_{}.txt'.format(
+            file_name, year, month))
+
+        if os.path.isfile(file_path):
+            file_paths_list.append(file_path)
+
+        data_obj = FileReader()
+        if len(file_paths_list) == 0:
+            print("No record exists for this year, Try another year!")
+        else:
+            # filled the data of the year give
+            data_obj.data_filler(file_paths_list)
+
+            # list for only max temperature of all tha days of the year
+            highest_temp_list = []
+            highest_temp_list = data_obj.max_temperature_c
+
+            # list for only min temperature of all tha days of the year
+            lowest_temp_list = []
+            lowest_temp_list = data_obj.min_temperature_c
+
+            result_calculator_obj = ResultCaluculator()
+            result_calculator_obj.draw_monthly_temperature_chart_bonus_task(
+                highest_temp_list, lowest_temp_list)
+
+            print(result_calculator_obj.monthly_temperature_chart)
```

Comment:
Add new line at bottom of file.

Reply:
missed it 
I wast formatting the project time to time
I will do all changings in weather man after meeting with u